Documentation/gettingstarted/k8s-install-rke.rst

```diff
@@ -0,0 +1,131 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _k8s_install_rke:
+
+********************************************
+Installation using Rancher Kubernetes Engine
+********************************************
+
+The guide walks you through integrating Cilium CNI with Kubernetes clusters
+deployed using Rancher Kubernetes Engine (RKE). The guide uses:
+
+  - Rancher Kubernetes Engine v1.0.8
+  - `Helm`_ Version 3
+
+Please consult `RKE Requirements`_ and Cilium :ref:`admin_system_reqs`.
+
+.. _RKE Requirements: https://rancher.com/docs/rke/latest/en/os/
+.. _Helm: https://helm.sh
+
+Installing Rancher Kubernetes Engine
+====================================
+
+Install the RKE Binary according to the `RKE Installation Guide <https://rancher.com/docs/rke/latest/en/installation/>`_
+
+Alternatively, on macOS the binary can be installed using Homebrew:
+
+.. code:: bash
+
+  $ brew install rke
+
+If you have a previous version of RKE installed, you can update it:
+
+
+.. code:: bash
+
+  $ brew upgrade rke
+
+Prepare the Kubernetes Cluster
+==============================
+
+Prepare your hosts with a supported operating system and Docker runtime. This
+is outside the scope of this guide.
```

Comment:
Makes sense.  Is there a link to RKE docs that cover this?    

For example, ideally we could point users who are new to Rancher (like me!) at https://rancher.com/docs/rancher/v2.x/en/quick-start-guide/deployment/quickstart-vagrant/

However, if one just follows the vagrant guide, they will get a setup with canal.  I had tried adding "network: none" to the config.yaml, but that config.yaml seems to be totally different than the cluster.yaml mentioned below.  Instead, I had to edit scripts/configure_rancher_server.sh to set network plugin to none, and that seemed to work.   It could be worth documenting this specifically, as it let's any newbie like me run the guide quite easily.

Reply:
Right, this is also the documentation for Rancher Server, which is a different product completely so providing a link to the correct documentation is clearly helpful.

Reply:
@seanmwinn you mentioned that providing a link is helpful, but this thread is marked as resolved and there is no link. Are you planning to add some suggested links here? (Marking unresolved for visibility, feel free to resolve when addressed)